xmake/modules/package/manager/clib/install_package.lua

```diff
@@ -0,0 +1,80 @@
+--!A cross-platform build utility based on Lua
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+-- 
+-- Copyright (C) 2015 - 2019, TBOOX Open Source Group.
+--
+-- @author      Adel Vilkov (aka RaZeR-RBI)
+-- @file        install_package.lua
+--
+
+-- imports
+import("core.base.option")
+import("core.project.config")
+import("lib.detect.find_tool")
+-- install package
+-- @param name  the package name, e.g. clib::clibs/bytes@0.4.0
+-- @param opt   the options, .e.g { verbose = true, out_dir = "clib",
+--                                  save = false, save_dev = false }
+--
+-- @return      true or false
+--
+function main(name, opt)
+    -- find clib
+    local clib = find_tool("clib")
+    if not clib then
+        raise("clib not found!")
+    end
+
+    -- default options
+    local all_opts = {
+        verbose = true,
+        out_dir = "clib",
+        save = false,
+        save_dev = false
+    }
+    -- copy specified options
+    if opt then
+        for k, v in ipairs(opt) do
+            all_opts[k] = v
+        end
+    end
+
```

Comment:
Please add `configurations()` function to define options and their default values in `install_package.lua`

https://github.com/xmake-io/xmake/blob/85cb688e30f996c5ac1036ec99b6c7c1a45cb035/xmake/modules/package/manager/conan/install_package.lua#L86-L96